infrastructure/files/game/cloudbuild.yaml.tpl

```diff
@@ -0,0 +1,87 @@
+# Copyright 2023 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+serviceAccount: projects/$${PROJECT_ID}/serviceAccounts/cloudbuild-cicd@$${PROJECT_ID}.iam.gserviceaccount.com
+steps:
+
+  #
+  # Building of the images
+  #
+
+  # Login to Github to be able to access Unreal container
+  - name: gcr.io/cloud-builders/docker
+    id: github-login
+    script: |
+      echo $CR_PAT | docker login ghcr.io -u USERNAME --password-stdin
+    secretEnv:
+      - CR_PAT
+
+  # Build Linux Client & Server Binaries using UE4 container
+  - name: ghcr.io/epicgames/unreal-engine:dev-slim-5.1.0
+    id: build-linux-binaries
+    args: [ "/home/ue4/UnrealEngine/Engine/Build/BatchFiles/RunUAT.sh", "BuildCookRun", 
+            "--Server", "-ServerConfig=Development",
+            "-Project=/workspace/Droidshooter.uproject", "-UTF8Output", "-NoDebugInfo", "-AllMaps",
```

Comment:
oh yeah! `/workspace/Droidshooter.uproject` that makes more sense, since we're pushing that cloudbuild.yaml up 👍🏻